libs/net/template/ReportExtensions.cs

```diff
@@ -268,12 +268,17 @@ public static string GetContentGroupByPropertyValue(this ContentModel content, s
     /// <param name="contentList"></param>
     /// <returns></returns>
     public static Dictionary<string, int> GetTotalScoresByTopicType(this IEnumerable<ContentModel> contentList) {
+        
         return contentList
-            .Where(x => x.Topics.All(a => a.Name != "Not Applicable"))
+            .Where(a => 
+                a.Topics.Any() // must have at least ONE Topic set
+                && a.Topics.All(a => a.Name != "Not Applicable" && !string.IsNullOrEmpty(a.Name)) // Name must not "Not Applicable" OR be empty
+                && a.Topics.All(t => t.Score > 0) // must have a Topic Score > ZERO
+            )
             .GroupBy(g => g.GetContentGroupByPropertyValue("topicType"))
             .ToDictionary(
                 g => g.Key,
-                g => g.ToList().Sum(s => s.Topics.FirstOrDefault()?.Score ?? 0)
+                g => g.Sum(s => s.Topics.First().Score)
```

Comment:
Won't this be more breakable than `FirstOrDefault() ?? 0`?

Reply:
The filtering logic above it should remove any instances of content that has no `Topic`